examples/iscsi/README.md

```diff
@@ -1,15 +1,16 @@
-# How to Use it?
-Here is my setup to setup Kubernetes with iSCSI persistent storage. I use Fedora 21 on Kubernetes node. 
-
-Install iSCSI initiator on the node:
+## Step 1. Setting up iSCSI target and iSCSI initiator
+I use Fedora 21 on Kubernetes node. First install iSCSI initiator on the node:
 
     # yum -y install iscsi-initiator-utils
    
    
 then edit */etc/iscsi/initiatorname.iscsi* and */etc/iscsi/iscsid.conf* to match your iSCSI target configuration.
 
-I mostly follow these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi&f=2) to setup iSCSI initiator and these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi) to setup iSCSI target.
+I mostly followed these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi&f=2) to setup iSCSI initiator and these [instructions](http://www.server-world.info/en/note?os=Fedora_21&p=iscsi) to setup iSCSI target.
+
+[Here](https://github.com/caesarxuchao/iscsiOnGCE) is an alternative setup on GCE.
```

Comment:
@rootfs thinks that should be kept elsewhere. I will add the instructions back to the README and try to make it concise.